src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -152,6 +152,12 @@ public function prepend(ContainerBuilder $container)
         }
 
         if ($container->hasExtension('doctrine')) {
+            // depending on the gedmo version it is in gedmo lib or src folder
+            // we use the autoloader to detect the correct directory
+            // see also: https://github.com/sulu/sulu/pull/5753
+            $reflection = new \ReflectionClass(\Gedmo\Exception::class);
+            $gedmoDirectory = \dirname($reflection->getFileName());
```

Comment:
using the autoloader to detect the file place fixes also the problem that every bundle need to set the gedmo extension path itself as for example here: https://github.com/sulu/SuluFormBundle/blob/5e353a7bb37be000c6c4e4733c8a938bffdb2710/Tests/Application/config/config.yml#L9-L17

Reply:
🎉 🎉 🎉